### ISSUE TITLE
Bug: No menu in RIDE4 on OSX

### ISSUE THREAD

**Issue by fourier** [Dyalog/ride#17]
```Version: 4.0.2559
  Platform: MacIntel
  Date: 2016-12-19 09:16:32 +0000
  Git commit: b07720323cca4239e3e761c5daecbc1aa9045bbf
```
The RIDE built winh **node mk dist** doesn't have a menu on OSX:
<img width="901" alt="screen shot 2017-01-03 at 20 39 33" src="https://cloud.githubusercontent.com/assets/900250/21620489/e3222364-d1f4-11e6-9414-e10a1e687fd6.png">

**Comment by jayfoad** [comment]
See http://stackoverflow.com/a/40445522

It looks like RIDE did once have some (commented out) code calling Menu.setApplicationMenu: ba402b6
But it was later removed: 1cae206

**Comment by jayfoad** [comment]
Fixed by 2b01969378d9c1b15f42d09bf60c95273a31eb2b

**Comment by fourier** [comment]
Thanks, look like the menu is present.
